LoopKitUI/Extensions/OrientationLock.swift

```diff
@@ -11,27 +11,29 @@ import SwiftUI
 
 public protocol DeviceOrientationController: AnyObject {
     var supportedInterfaceOrientations: UIInterfaceOrientationMask { get set }
+    func setOriginallySupportedInferfaceOrientations()
 }
 
 /// A class whose lifetime defines the application's supported interface orientations.
```

Comment:
[nit] It's now longer the `class` lifetime that determines the locking, right?  Please update the comment to reflect...